tools/plotly_regression_performance_plots/plotly_regression_performance_plots.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="plotly_regression_performance_plots" name="Plot actual vs predicted curves and residual plots" version="0.1">
+    <description>of tabular data</description>
+    <requirements>
+        <requirement type="package" version="3.6">python</requirement>
+        <requirement type="package" version="0.22.0">pandas</requirement>
+        <requirement type="package" version="3.1.1">plotly</requirement>
+    </requirements>
+    <version_command>echo $version</version_command>
+    <command detect_errors="aggressive"><![CDATA[
+    python '$__tool_directory__/plot_regression_performance.py'
+        -i '$infile_input'
+        -j '$infile_output'
+]]>
+    </command>
+    <inputs>
+        <param name="infile_input" type="data" format="tabular" label="Select input data file :" help="Input data is a matrix (tabular) where each column is a feature and the last column contains the target values."/>
+        <param name="infile_output" type="data" format="tabular" label="Select predicted data file :" help="Predicted data is a matrix (tabular) where each column is a feature and the last column contains the predicted values."/>
```

Comment:
I think once the last column contains the designated values, the data can be retrieved for visualization. In other words, files contain one column which are the true labels are still valid input here.

Reply:
Yes, just true and predicted value columns are enough for inputs. But, we just get complete data either with true values or predicted values.

Reply:
How about this?
```A tabular in which the last column (including the column if there is only one) contains the target values ```

Reply:
Update the help tex. Thanks :)